### ISSUE TITLE
Test functionality of RADX MARS OTC Condition to Note Conversion feature

### ISSUE THREAD

**Issue by chris-kuryak** [CDCgov/prime-reportstream#14693]
## Problem statement
RADX MARS spec requires unwanted conditions to be converted to notes before sending to receiver.

## Suggestions to executing

- Create schema that converts conditions that a receiver does not want in multiplex results to a note per RadX Mars spec
- Setup Test receiver
- Test in staging or locally
- Confirm notes created properly
- Won't be able to use Condition Filter, have to remove unwanted conditions as a transform

## Acceptance criteria

- [ ] Conditions not wanted from receiver in multiplex results are removed
- [ ] Conditions not wanted from receiver in multiplex results are converted to notes in the message to a receiver per RADX MARS spec

## To do
- [x] Confirm with Platform team if there is a requirement around HOW the note(s) should look

**Comment by chris-kuryak** [comment]
From Platform (Brandon): 
No specifics on how the NTE should be formatted and just wanted to make sure it's preserved somewhere which to me sounds like copy/paste logic from the OBX segment to an NTE

**Comment by chris-kuryak** [comment]
Updated guidance from Platform with some examples:

NTE|1|O|OBX filtered for identifier = 92142-9 - Influenza virus A RNA [Presence] in Respiratory specimen by NAA with probe detection; value = 260385009 - Negative. OBX was removed due to your jurisdictional reporting rules indicating this result is not reportable.

NTE|2|O|OBX filtered for identifier = 92141-1 - Influenza virus B RNA [Presence] in Respiratory specimen by NAA with probe detection; value = 260385009 - Negative. OBX was removed due to your jurisdictional reporting rules indicating this result is not reportable.

NTE|3|O|OBX filtered for identifier = 92131-2 - Respiratory syncytial virus RNA [Presence] in Respiratory specimen by NAA with probe detection; value = 260385009 - Negative. OBX was removed due to your jurisdictional reporting rules indicating this result is not reportable.

**Comment by chris-kuryak** [comment]
Waiting on Platform to finish #14924 before we can finish testing

**Comment by chris-kuryak** [comment]
This is apparently now unblocked per Platform telling us they have finished and merged their changes.

**Comment by JessicaWNava** [comment]
They created a new ticket for this  https://github.com/CDCgov/prime-reportstream/issues/15169 and this is still open and there is not a closed PR for it, so this is still not fixed. Moving back to Waiting for Confirmation.

**Comment by chris-kuryak** [comment]
Platform has indicated that they will finish up 15169 by end of August. So we can pick this ticket back up in September.

**Comment by chris-kuryak** [comment]
Platform has indicated that Engagement should try to develop a near term solution using a FHIR to HL7 transform, as they work on a more longer term solution for a FHIR to FHIR transform.

Chris and Victor to put a ticket on Engagement board to create a FHIR to HL7 transform AND unblock this ticket.

**Comment by chris-kuryak** [comment]
Should be unblocked now

**Comment by JessicaWNava** [comment]
Lucero fixed this as part of  https://github.com/CDCgov/prime-reportstream/pull/15764
